pkgs/applications/graphics/textual-paint/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, python3
+, fetchFromGitHub
+, fetchPypi
+, fetchpatch
+}:
+
+let
+  python = python3.override {
+    packageOverrides = _: super: {
+      pillow = super.pillow.overridePythonAttrs rec {
+        version = "9.5.0";
```

Comment:
There are other applications that call for older `pillow` versions. For example, I [proposed adding a patch](https://github.com/NixOS/nixpkgs/pull/247618/files#diff-abf89f4398bd57d38105975edb6f1aaf6671b588eadedb84f6dd7b4f59a4bfb2) to another application to point it at `10.0.0`. Would it be better to add a `pillow_950` to `python3Packages`?

Reply:
I wouldn't put it in this PR, since it is already fairly large, also the package you are working on seems to work after relaxing the required version, but textual-paint doesn't.